src/main/java/rs117/hd/HdPluginConfig.java

```diff
@@ -175,7 +175,7 @@ default SyncMode syncMode()
 			section = generalSettings
 	)
 	@Range(
-			min = 1,
+			min = 0,
```

Comment:
Did you mean to reduce this minimum to zero? This is my only concern with these changes, otherwise LGTM :)

Reply:
Yeah - it's basically just a checkbox to bypass the FPS limit but without taking up extra UI space.

Reply:
A value of zero or less simply skips the `Thread.sleep()` call as far as I can tell.